index.html

```diff
@@ -0,0 +1,550 @@
+</head>
+
+<body>
+  <header class="bg-gray-1">
+    <div>
+      <nav class="nav">
+        <ul class="nav-ul">
+          <li class="nav-items">
+            <a href="#" class="brand-logo">
+              <img src="https://d33wubrfki0l68.cloudfront.net/81fa290218cedc39d32be7b1f1aba58a25829ebe/0a05f/images/logo--full.svg" alt="Smashing Magazine" />
+            </a>
+          </li>
+          <li class="nav-items">
+            <p>Article</p>
+            <div>Design & development</div>
+          </li>
+          <li class="nav-items">
+            <p>Books</p>
+            <div>Physical & digital books</div>
+          </li>
+          <li class="nav-items">
+            <p>Events</p>
+            <div>Conferences & workshops</div>
+          </li>
+          <li class="nav-items">
+            <p>Jobs</p>
+            <div>Find work & employees</div>
+          </li>
+          <li class="nav-items">
+            <p>Membership</p>
+            <div>Webinars & early-birds</div>
+          </li>
+        </ul>
+        <button class="btn-nav"><span></span>Topics</button>
+      </nav>
+    </div>
+  </header>
+  <main>
+    <!-- employee comments -->
+    <section class="featured-articles bg-gray-2">
+
+      <div class="employee-comments">
+        <article class="first-div">
+          <h2>philp kiely comment</h2>
+          <p>Philip Kiely <span>wrote</span></p>
+          <h3>Django Highlight: User Models and Authentication (Part 1)</h3>
+          <span><i class="fab fa-comment-alt fa-2x fa-fw"></i>Leave a
+            comment</span>
+          <div class="first-div-floats">
+            <p># Apps</p>
+            <p># Frameworks</p>
+          </div>
+        </article>
+        <article class="middle-div bg-gray-3">
+          <h2>dan halliday comment</h2>
+          <h3 class="section-title"><span>Don't Miss These Latest<br>Articles</span></h3>
+          <p>Dan Halliday <span>wrote</span></p>
+          <h3>Magic flip cards solving a common sizing problem</h3><span><i class="fab fa-comment-alt fa-2x fa-fw"></i> 3 comments</span>
+          <div class="first-div-floats">
+            <p># CSS</p>
+            <p># User Interaction</p>
+          </div>
+        </article>
+
+        <article class="end-div">
+          <h2>Sarah Drasner & Geoff Graham comment</h2>
+          <div class="frame">
+            <div class="frame-img"></div>
+            <button>Seriously, red?</button>
+          </div>
+          <p>Sarah Drasner & Geoff Graham <span>wrote</span></p>
+          <h3>How To Create A Headless WordPress Site On The JAMstack</h3>
+          <span><i class="fab fa-comment-alt fa-2x fa-fw"></i> 14 comments</span>
+          <div class="first-div-floats">
+            <p># Wordpress</p>
+            <p># Static Generators</p>
+          </div>
+        </article>
+      </div>
+    </section>
+    <!-- latest post -->
+    <section>
+      <h2>latest post</h2>
+      <div class="latest-post bg-gray-1">
+        <div class="md">
```

Comment:
line #95 should be `<article>` and not a `<div>`

Reply:
done